docs/node_setup.md

```diff
@@ -1,127 +1,122 @@
 ---
 id: node_setup
-title: Setup validator node
+title: Run a validator node
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-## Initial setup
+This section walks you through the process of setting up and running a HydraDX validator node.
```

Comment:
Maybe we should add warning that it's technical you need guaranteed uptime and could lead to loss of funds for you and nominators if done wrong. And ppl are not sure they should nominate instead.

Reply:
good idea!